<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaComodin</class>
 <widget class="QDialog" name="ventanaComodin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/imagenes/blanca.png);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>351</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Valor del comodin</string>
   </property>
   <widget class="QRadioButton" name="botonA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonF">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonG">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonH">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonI">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>I</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonJ">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>J</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonK">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonL">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>L</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonM">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonN">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonO">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonP">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonQ">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Q</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonR">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonS">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonT">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>T</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonU">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>U</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonV">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonW">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonX">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonY">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="botonZ">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonSeleccionar">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
